testProjects/packageJsonConfig/handler.test.js

```diff
@@ -0,0 +1,25 @@
+import _ from 'lodash';
+
+const iopipe = require('./iopipe');
+
+describe('Using package.json iopipe configuration', () => {
+  beforeEach(() => {
+    delete process.env.IOPIPE_TOKEN;
```

Comment:
an enhancement to this (not necessarily this PR to the branch) would be to add a test covering the order preference (ex. when env var is set and config things are set and some code things are set etc.)